src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/Helpers/iOS/Extensions/UIViewExtensions.ios.cs

```diff
@@ -4,54 +4,24 @@ namespace Xamarin.CommunityToolkit.UI.Views.Helpers.iOS.Extensions
 {
 	static class UIViewExtensions
 	{
-		public static NSLayoutYAxisAnchor SafeBottomAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.BottomAnchor
-				: view.BottomAnchor;
-
-		public static NSLayoutXAxisAnchor SafeCenterXAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.CenterXAnchor
-				: view.CenterXAnchor;
-
-		public static NSLayoutYAxisAnchor SafeCenterYAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.CenterYAnchor
-				: view.CenterYAnchor;
-
-		public static NSLayoutDimension SafeHeightAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.HeightAnchor
-				: view.HeightAnchor;
-
-		public static NSLayoutXAxisAnchor SafeLeadingAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.LeadingAnchor
-				: view.LeadingAnchor;
-
-		public static NSLayoutXAxisAnchor SafeLeftAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.LeftAnchor
-				: view.LeftAnchor;
-
-		public static NSLayoutXAxisAnchor SafeRightAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.RightAnchor
-				: view.RightAnchor;
-
-		public static NSLayoutYAxisAnchor SafeTopAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.TopAnchor
-				: view.TopAnchor;
-
-		public static NSLayoutXAxisAnchor SafeTrailingAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.TrailingAnchor
-				: view.TrailingAnchor;
-
-		public static NSLayoutDimension SafeWidthAnchor(this UIView view) =>
-			UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
-				? view.SafeAreaLayoutGuide.WidthAnchor
-				: view.WidthAnchor;
+		public static NSLayoutYAxisAnchor SafeBottomAnchor(this UIView view) => view.SafeAreaLayoutGuide.BottomAnchor;
```

Comment:
Why did we drop the system version check?)

Reply:
I removed parentController(which used version below 11) and replaced it with Anchor. Do you have a chance to check the app on iOS 10?
As far as I know Apple requires minimum version as iOS 13 to publish new apps. Should Maui and Xamarin support previous versions?

Reply:
I guess we should preserve the iOS version check.

https://developer.apple.com/documentation/uikit/uiview/2891102-safearealayoutguide

It will crash on iOS 10

Reply:
ok, I will revert this checks

Reply:
fixed